cypress/e2e/shared/checks.test.ts

```diff
@@ -519,6 +519,14 @@ describe('Checks', () => {
     })
 
     it('can edit the check card', () => {
+      const newDescription =
+        'русский дворянин, лечившийся четыре года в Швейцарии от эпилепсии.'
+      const newName = 'Князь Мышкин'
```

Comment:
could we change this to english? 😅

Reply:
Yes, I'll change this to english.  In the past I've worked on projects where internationalization was important, so I developed a habit of setting up test values that use non-latin unicode sets.  But, I think this is covered in other tests.